src/main/java/com/systelab/seed/controller/PatientController.java

```diff
@@ -32,40 +47,70 @@ public PatientController(PatientService patientService) {
         this.patientService = patientService;
     }
 
-    @ApiOperation(value = "Get all Patients", authorizations = {@Authorization(value = "Bearer")})
+    @ApiOperation(value = "Get all Patients", authorizations = { @Authorization(value = "Bearer") })
     @GetMapping("patients")
     public ResponseEntity<Page<Patient>> getAllPatients(Pageable pageable) {
         return ResponseEntity.ok(this.patientService.getAllPatients(pageable));
     }
 
-    @ApiOperation(value = "Get Patient", authorizations = {@Authorization(value = "Bearer")})
+    @ApiOperation(value = "Get Patient", authorizations = { @Authorization(value = "Bearer") })
     @GetMapping("patients/{uid}")
     public ResponseEntity<Patient> getPatient(@PathVariable("uid") UUID id) {
         return ResponseEntity.ok(this.patientService.getPatient(id));
     }
 
-    @ApiOperation(value = "Create a Patient", authorizations = {@Authorization(value = "Bearer")})
+    @ApiOperation(value = "Create a Patient", authorizations = { @Authorization(value = "Bearer") })
     @PostMapping("patients/patient")
     public ResponseEntity<Patient> createPatient(@RequestBody @ApiParam(value = "Patient", required = true) @Valid Patient p) {
         Patient patient = this.patientService.createPatient(p);
         URI uri = MvcUriComponentsBuilder.fromController(getClass()).path("/patients/{id}").buildAndExpand(patient.getId()).toUri();
         return ResponseEntity.created(uri).body(patient);
     }
 
-
-    @ApiOperation(value = "Create or Update (idempotent) an existing Patient", authorizations = {@Authorization(value = "Bearer")})
+    @ApiOperation(value = "Create or Update (idempotent) an existing Patient", authorizations = { @Authorization(value = "Bearer") })
     @PutMapping("patients/{uid}")
     public ResponseEntity<Patient> updatePatient(@PathVariable("uid") UUID id, @RequestBody @ApiParam(value = "Patient", required = true) @Valid Patient p) {
         Patient patient = this.patientService.updatePatient(id, p);
         URI selfLink = URI.create(ServletUriComponentsBuilder.fromCurrentRequest().toUriString());
         return ResponseEntity.created(selfLink).body(patient);
     }
 
+    @ApiOperation(value = "Add an alergy to a Patient", authorizations = { @Authorization(value = "Bearer") })
+    @PutMapping("patients/{patientUid}/allergies/{allergyUid}")
+    public ResponseEntity<Patient> addAlergyToPatient(@PathVariable("patientUid") UUID patientId, @PathVariable("allergyUid") UUID allergyId,
```

Comment:
In the specification, two operations are specified, by POST and by PUT. It is important to stick to the specification in order to work with the frontend.

Reply:
Operations added as mentioned in the specification